internal/provider/waypoint/resource_waypoint_add_on_test.go

```diff
@@ -228,6 +230,7 @@ resource "hcp_waypoint_template" "test" {
   summary = "some summary for fun"
   readme_markdown_template = base64encode("# Some Readme")
   terraform_no_code_module_source = "private/waypoint-tfc-testing/waypoint-template-starter/null"
+	terraform_project_id = "prj-gfVyPJ2q2Aurn25o"
```

Comment:
This entire file is a mix of tabs and spaces. I agree. Ideally this would happen before these tests were merged

Reply:
I tried to fix some of them in this PR, which is the changes in this test file you see that aren't related to `terraform_project_id`

Reply:
> . Ideally this would happen before these tests were merged

I am not sure what it means since you want to merge in main? I expected the fix to be done in the scope of this PR.

Reply:
Fixed in ead3fc328468fbbc1ad47abc128c760994222aca

And what I meant was, these tests existed before this PR was open, and it has plenty of mixed spacing and tabs:
* https://github.com/hashicorp/terraform-provider-hcp/pull/1052/files#diff-6535482890df9ededf38feccdf323552d43d8c7f50d4c443d9a45c9e811907e8L255-L354
* https://github.com/hashicorp/terraform-provider-hcp/pull/1052/files#diff-01be39748de68db6001914f74e3b360072065eb46d86f41a0c3fa9c555ebbe6cL283-L295

They make PR linters for things like this so next time it would be nice if that was in place!